README.md

```diff
@@ -12,12 +12,31 @@ A list of awesome [RemNote](https://remnote.io) plugins, themes, tutorials and o
  such that the devs can focus on RemNote itself. The builtin tutorials
  are great but there are some outdated and missing parts.
  -->
-
+ <!-- Add contributing guidelines -->
+- [Awesome Remnote](#awesome-remnote)
+  - [Community](#community)
+  - [Integrations](#integrations)
+  - [Extensions](#extensions)
+    - [Plugins](#plugins)
+    - [Themes](#themes)
+    - [Custom CSS Mixins](#custom-css-mixins)
+    - [Development](#development)
+  - [Tutorials](#tutorials)
+    - [Video Tutorials](#video-tutorials)
+    - [Courses](#courses)
+- [Education Materials](#education-materials)
+  - [Learning](#learning)
+    - [Spaced Repetition](#spaced-repetition)
+    - [Note-taking](#note-taking)
+  - [Knowledge Bases](#knowledge-bases)
```

Comment:
I like your reorganization. I am not completely satisfied with the Education Materials section. Maybe flatten it out a bit more (flat hierarchies are good I heard) and factor out Knowledge Bases because it is not really education?:

- [Education Materials](#education-materials)
  - [Learning](#learning)
  - [Spaced Repetition](#spaced-repetition) (or leave this indented?)
  - [Note-taking](#note-taking)
- [Knowledge Bases](#knowledge-bases)

Btw. what is your rationale to rename Writing to Note-taking? For me note-taking has a slight connotation of "just" repeating ideas (already distilled in a book etc.) in your own words. I feel Writing expresses the creative part to come up with your own ideas more explicitly? Or is it that note-taking is the more common term in the student community?

Reply:
Personally I prefer to keep things deeply outlined and collapsible, but there is no way to collapse sections in markdown, so you may have to make a decision on how flat you want things to be.

Elaboration is a part of note-taking. You elaborate by the act of putting hands on writing implements, yes, but "writing" has a connotation of being creative writing or editing tips. IMO note-taking is well in scope for RemNote, while penning manuscripts is possibly a little too offtopic.

Reply:
It probably takes just a bit for me to get used to the term note-taking.
And heading structure is no big deal. It's just big bold text after all :p